.github/workflows/install_dependencies_ubuntu.sh

```diff
@@ -9,6 +9,11 @@ apt-get install -y \
     dejagnu \
     iverilog
 
-if [ `lsb_release -rs` != 16.04 ]; then
+REL = `lsb_release -rs | tr -d .`
```

Comment:
This line is failing in the CI:
```
.github/workflows/install_dependencies_ubuntu.sh: line 12: REL: command not found
```
This doesn't cause the CI to fail -- it probably should? -- but it causes the script to end at this point, without installing either lld or systemc.  So then the testsuite runs fail when it needs systemc (on Ubuntu 20.04) and when it uses the linker (on Ubuntu 18.04 and 20.04).  (The places where it fails because of the linker is just the handful of imported-C tests.  Are you expecting that the linker gets used more than that?)

Reply:
I'm an idiot! I fixed this before too, but somehow the changes were lost. Unlike in make, the whitespace is significant here. `REL=foo` is OK, but `REL = foo` is not!